requests_wrapper.py

```diff
@@ -0,0 +1,43 @@
+import requests
```

Comment:
I'm almost certain this isn't useful.  We have a whole extra class because someone didn't want to have to deal with timeouts and setting auth? 
I know you didn't write it but can you change the notify gateway to use the timeout and auth args and remove this class?